python/torch_mlir/extras/fx_importer.py

```diff
@@ -479,14 +483,18 @@ def format_asm_shape(self, shape: torch.Size) -> str:
     """Return IrType for !torch.vtensor with the given shape and dtype"""
 
     def get_vtensor_type(
-        self, shape: torch.Size, dtype: torch.dtype, sparse_layout: torch.layout = None
+        self,
+        shape: torch.Size,
+        dtype: torch.dtype,
+        sparsity: Tuple[torch.layout, int, int],
     ):
         shape_asm = self.format_asm_shape(shape)
         mlir_dtype = str(self.dtype_to_type(dtype))
-        if sparse_layout is not None:
-            sparsity = sparsity_encoding(shape, sparse_layout)
+        if sparsity is not None:
+            sparse_layout, posw, crdw = sparsity
```

Comment:
Is `nnz` also needed here?

Reply:
most likely, also sparsity level eventually (since CSR/CSC can be define at higher dimension in torch.sparse).

but for the 2-dim CSR/CSC, this is all we need (with probably _**two**_ more ints at most in a future extension)